### PR TITLE
Make Plasma Fist less costly - From 4 uses to 1

### PR DESCRIPTION
**Pull request by SplergeBucket**
fuck you crystal you git
this makes plasma fist a more viable option, right now it costs twice as much as some of the best spells in the game

### CODE REVIEW

**Comment by SplergeBucket** [comment]
aw ye baby's first PR

**Comment by danhibiki** [comment]
:+1:

File: code/game/gamemodes/wizard/spellbook.dm [hunk 1]
```diff
@@ -347,7 +347,7 @@
 	desc = "Consider this more of a \"Spell Bundle\". This artifact is NOT reccomended for weaklings. An ancient scroll that will teach you the art of Plasma Fist. With it's various combos you can knock people down in the area around you, light them on fire and finally perform the PLASMA FIST that will gib your target."
 	item_path = /obj/item/weapon/plasma_fist_scroll
 	log_name = "PF"
-	cost = 4  //VERY costly. Can only buy lvl1 magic missile/fireball/etc.
+	cost = 1  //It's a combination of kickass fighting moves, which will often require magic missile or some other disabling spell to work well.
```
Comment (nalarmg) [inline code/game/gamemodes/wizard/spellbook.dm]: What's the purpose of this comment?

Comment (danhibiki) [inline code/game/gamemodes/wizard/spellbook.dm]: I suppose so its just not improperly reflecting it, although it obviously has a personal touch. If he lowered the cost to 1, and it still said "VERY COSTLY", looks a little silly to anyone reading.

Comment (nalarmg) [inline code/game/gamemodes/wizard/spellbook.dm]: Doesn't really seem to serve a purpose. It's just there.

Comment (SplergeBucket) [inline code/game/gamemodes/wizard/spellbook.dm]: iunno, i just wanted to git rid of the "very costly"

**Comment by Crystalwarrior** [comment]
:+1:

**Comment by Nebyab** [comment]
sure

**Comment by Lawligagger** [comment]
Alright.

**Pull request merged**